app/templates/views/confirm_email_address.html

```diff
@@ -0,0 +1,42 @@
+{% extends "document_download_template.html" %}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+
+
+{% block per_page_title %}
+  Confirm your email address
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if show_invalid_email_error %}
+        {{ govukErrorSummary({
+          "titleText": "There is a problem",
+          "errorList": [
+            {
+              "text": "Not a valid email address",
+              "href": "#email-address-input"
+            },
+          ]
+        }) }}
+      {% endif %}
+
+      <h1 class="govuk-heading-l">Confirm your email address</h1>
+
+      <p class="govuk-body">
```

Comment:
Think this is gonna need to be included in the `EmailAddress` form field as the `hint` param instead. See what it looks like but if you need it to be this colour and have this spacing, the `hint` param lets us add classes so it shouldn't be too hard.

Reply:
We're not going to wrap the h1 in the label for now (see [this comment](https://github.com/alphagov/document-download-frontend/pull/129#discussion_r960589626)) so this comment is redundant.